lib/components/CelebGallery/index.tsx

```diff
@@ -1,52 +1,56 @@
 import React from 'react';
 import Image from 'next/image';
 import { sanityImage } from '~/lib/components/sanityio';
-import { Typography } from '@mui/material';
-import s from './styles.module.scss';
 import { TCelebGalleryItem } from '~/lib/components/types';
 import { useCelebContext } from '~/lib/components/StaticPropsContextProvider';
+import {useRouter} from 'next/router';
 
 export const CelebGallery: React.FC<{
   celebGalleryItems: TCelebGalleryItem[];
 }> = (p) => {
   const context = useCelebContext();
+  const router = useRouter()
 
+  // dynamic classes
+  let grid_class = ['grid', 'gap-4', router.pathname == '/' ? 'grid-cols-2' : 'grid-cols-3', router.pathname == '/' ? 'sm:grid-cols-3' : 'sm:grid-cols-4', 'lg:grid-cols-4', 'items-center', 'justify-center', 'w-full'].join(' ')
   return (
-    <div className={s.CelebGallery}>
-      {p.celebGalleryItems.map((celebData) => {
-        const picture = celebData.picture || context.placeholderImage;
+    <div className="container mx-auto w-full h-auto items-center justify-center">
+      <div className={grid_class}>
+        {p.celebGalleryItems.map((celebData) => {
+          const picture = celebData.picture || context.placeholderImage;
 
-        return (
-          <a
-            className={s.link}
-            href={`/${celebData.slug}`}
-            key={celebData.slug}
-          >
-            <span className={s.image}>
-              <Image
-                objectFit="cover"
-                objectPosition="top"
-                blurDataURL={picture.metadata.lqip}
-                placeholder="blur"
-                src={sanityImage(picture).width(200).height(250).url()}
-                alt={celebData.name}
-                layout="fixed"
-                width={160}
-                height={200}
-                className={s.image}
-              />
-            </span>
-            <Typography
-              fontWeight={500}
-              variant="h4"
-              component="p"
-              className={s.name}
+          return (
+            <a
+              className="celeb-card overflow-hidden rounded-xl w-full"
```

Comment:
is it possible to migrate this `celeb-card` class to Tailwind?

Reply:
I'm using Tailwind CSS in this class `celeb-card` but I don't think we can achieve this animation if we put the Tailwind CSS in the HTML tag. We can't directly access the other elements in Tailwind CSS Classes.<br>
![image](https://user-images.githubusercontent.com/68635840/165448190-a9a47760-d4e8-43c7-b3cc-6ff340d22e5d.png)
![screen-capture](https://user-images.githubusercontent.com/68635840/165448501-1dbeec2a-3c8c-4972-9b9f-f5561d7df065.gif)